### DISCUSSION TITLE
Rivatuner OSD toggle not working on latest Nightly Build (Mar 25, 2023)

### DISCUSSION THREAD

**Discussion by mrallroy** [LizardByte/community#232] [Sunshine] [answered]
As the title says, I am no longer able to toggle the OSD on or off in Rivatuner since upgrading to the latest nightly build (as of Mar 25, 2023).  I've confirmed this isn't an issue with both versions 18.4 and 19.1.  I've not made any setting changes in RT or AB and am running the latest versions of both.  I connect to sunshine via Shield TV w/ Moonlight.  My keyboard is a logitech K400+ that is connected to the Shield TV.  I've made no settings changes in both Shield TV and Moonlight.  To test, I simply downgrade from Nightly to 18.4 then upgrade to 19.1, reboot and the toggle works with both versions.  I've not used any other nightly builds so I can't confirm if this issue is new to the latest build or not.

Anyways, I thought I should post my findings for the devs and please let me know if there's any other information I may need to provide.  

Thank you.

**Comment by cgutman** [comment, marked as answer]
Filed #1233 to track this. We can continue discussion there.
